modules/migrating-sccs-during-the-manual-upgrade.adoc

```diff
@@ -0,0 +1,351 @@
++
+[source,terminal]
+----
+$ oc -n stackrox create -f ./update-central.yaml
+----
+
+* To add the required roles and role bindings to use the stock {osp} SCCs for all secured clusters, perform the following steps:
+
+.. Create a file named `upgrade-scs.yaml` that defines the role and role binding resources by using the following content:
++
+.Example YAML file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role  # <1>
+metadata:
+  annotations:
+     email: support@stackrox.com
+     owner: stackrox
+  labels:
+     app.kubernetes.io/component: collector
+     app.kubernetes.io/instance: stackrox-secured-cluster-services
+     app.kubernetes.io/name: stackrox
+     app.kubernetes.io/part-of: stackrox-secured-cluster-services
+     app.kubernetes.io/version: 4.4.0
+     auto-upgrade.stackrox.io/component: sensor
+  name: use-privileged-scc  # <2>
+  namespace: stackrox # <3> 
+rules:  # <4>
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - privileged
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+- - -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding # <5>
+metadata:
+  annotations:
+     email: support@stackrox.com
+     owner: stackrox
+  labels:
+     app.kubernetes.io/component: collector
+     app.kubernetes.io/instance: stackrox-secured-cluster-services
+     app.kubernetes.io/name: stackrox
+     app.kubernetes.io/part-of: stackrox-secured-cluster-services
+     app.kubernetes.io/version: 4.4.0
+     auto-upgrade.stackrox.io/component: sensor
+  name: collector-use-scc # <6>
+  namespace: stackrox
+roleRef: # <7>
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: use-privileged-scc
+subjects: # <8>
+- kind: ServiceAccount
+  name: collector
+  namespace: stackrox
+- - -
+----
+<1> The type of Kubernetes resource, in this example `Role`.
+<2> The name of the role resource.
+<3> The namespace in which the role is created. 
+<4> Describes the permissions granted by the role resource.
+<5> The type of Kubernetes resource, in this example, `RoleBinding`.
+<6> The name of the role binding resource.
+<7> Specifies the role to bind in the same namespace.
+<8> Specifies the subjects that are bound to the role.
+====
+
+.. Create the role and role binding resources specified in the `upgrade-scs.yaml` file by running the following command:
++
+[source,terminal]
+----
+$ oc -n stackrox create -f ./update-scs.yaml
+----
+
+. Delete the SCCs that are specific to {product-title-short}:
+
+* To delete the SCCs that are specific to Central, run the following command:
++
+[source,terminal]
+----
+$ oc delete scc/stackrox-central scc/stackrox-central-db scc/stackrox-scanner
+----
+
+* To delete the SCCs that are specific to all secured clusters, run the following command:
++
+[source,terminal]
+----
+$ oc delete scc/stackrox-admission-control scc/stackrox-collector scc/stackrox-sensor
```

Comment:
Same here, this command needs to be run on each cluster. I should have mentioned that in the initial document, apologies!